crates/core/tedge_mapper/src/c8y/service_monitor.rs

```diff
@@ -0,0 +1,157 @@
+use c8y_api::smartrest::topic::SMARTREST_PUBLISH_TOPIC;
+use mqtt_channel::Message;
+use mqtt_channel::Topic;
+use serde_json::json;
+use serde_json::Value;
+use std::collections::HashMap;
+
+pub fn convert_health_status_message(message: &Message, device_name: String) -> Vec<Message> {
+    let mut mqtt_messages: Vec<Message> = Vec::new();
+    let topic = message.topic.name.to_owned();
+    let service_name = get_service_name(&topic);
+    let child_id = get_child_id(&topic);
+
+    // If not Bridge health status
+    if !service_name.contains("mosquitto-c8y-bridge") {
```

Comment:
This check is too specific: there are also the `az` and `aws` bridge.

Reply:
Yes, it's for all the bridges. I changed to check for all the bridges.

Reply:
> Good point. An option could be to translate 1/0 as up/down. However not sure that this helpful as the point is to transfer this state over the bridge. @reubenmiller what do you think about this case?

Yes I would love to have a consistent health payload if possible, e.g. using `up/down` instead of `1/0`.

Reply:
> > Good point. An option could be to translate 1/0 as up/down. However not sure that this helpful as the point is to transfer this state over the bridge. @reubenmiller what do you think about this case?
> 
> Yes I would love to have a consistent health payload if possible, e.g. using `up/down` instead of `1/0`.

I think he meant that when we send it to c8y cloud. But do we really need the bridge status to be sent to the c8y cloud?. Because when the bridge goes down, the status can't be sent to the c8y cloud.